protostar/commands/init/project_creator/adapted_project_creator.py

```diff
@@ -1,12 +1,24 @@
 from pathlib import Path
 
-from protostar.cairo import CairoVersion
 from protostar.commands.init.project_creator._project_creator import ProjectCreator
+from protostar.configuration_file import ConfigurationFileV2Model
 
 
 class AdaptedProjectCreator(ProjectCreator):
     def run(self):
         project_root_path = Path()
-        self.save_protostar_toml(
-            project_root_path=project_root_path, cairo_version=CairoVersion.cairo0
+        config = ConfigurationFileV2Model(
+            protostar_version=str(self._protostar_version),
+            contract_name_to_path_strs={"main": ["src/main.cairo"]},
+            project_config={
+                "lib-path": "lib",
+                "linked-libraries": ["src"],
+            },
+            command_name_to_config={},
+            profile_name_to_project_config={},
+            profile_name_to_commands_config={},
```

Comment:
I noticed you do this in several places, maybe we could set default values for them in the `ConfigurationFileV2Model`?

Reply:
Good idea